cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1344,7 +1344,7 @@ class LoginNodesPoolSchema(BaseSchema):
     iam = fields.Nested(LoginNodesIamSchema, metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP})
     gracetime_period = fields.Int(
         validate=validate.Range(
-            min=1, max=120, error="The gracetime period for LoginNodes Pool must be an interger from 1 to 120."
+            min=3, max=120, error="The gracetime period for LoginNodes Pool must be an interger from 3 to 120."
```

Comment:
Typo: `interger` -> `integer`

Reply:
Thank you for your reminding, Eddy! Have corrected it. I will also create a PR for develop branch.